superset-websocket/src/index.ts

```diff
@@ -94,8 +94,17 @@ export const statsd = new StatsD({
 });
 
 // enforce JWT secret length
-if (startServer && opts.jwtSecret.length < 32)
-  throw new Error('Please provide a JWT secret at least 32 bytes long');
+if (startServer && opts.jwtSecret.length < 32) {
+  console.error('ERROR: Please provide a JWT secret at least 32 bytes long');
+  process.exit(1);
+}
+
+if (startServer && opts.jwtSecret.startsWith('CHANGE-ME')) {
+  console.warn(
+    'WARNING: it appears you secret in your config.json is insecure',
```

Comment:
On our first attempt to tackle the default `SECRET_KEY` issue, we did a similar thing on the Flask backend, and it was proven that many admins ignore these warnings even when the log warning is a large banner. 

Can we do it in such a way that is a specific config for docker-compose only? `docker/.env` holds many secret defaults but it's only used by docker-compose

Reply:
The way I did things here this file in the repo `./docker/superset-websocket/config.json` is only mounted in docker-compose. If you go raw docker or helm it wouldn't get picked up.

For clarity though now I wonder if we should create a new `docker/compose` folder for things meant only for docker-compose to use (?). Currently it's mostly resources for docker-compose, but some things might be reused in Dockerfile as entrypoints or by Helm(?)

Reply:
oh right, my comment was too early in the morning ;)

`docker/compose` seems like a good idea